LocalReferenceLinkRel/explainer.md

```diff
@@ -148,6 +148,79 @@ the shadow root where they are defined, as illustrated by the following examples
   Styles defined inside the sibling Shadow Root are not applied, so "Inside Sibling Shadow DOM" is not blue.
 </p>
 
+### Fetch Behavior
+
+The current behavior for the `<link>` tag with following markup in a file
```

Comment:
Suggested rephrasing: 

'In user agents where "Local References In `<link>` Tags" is not supported, the behavior for the `<link>` tag with the following markup in a file named "foo.html" will be as follows:'